packages/core/src/docdb/commands/createCluster.ts

```diff
@@ -34,6 +34,18 @@ export async function createCluster(node?: DocumentDBNode) {
     try {
         const cluster = await node?.createCluster(result)
 
+        // create instances for cluster
+        if (cluster && result.DBInstanceCount) {
+            for (let index = 0; index < result.DBInstanceCount; index++) {
+                await node?.client.createInstance({
+                    Engine: 'docdb',
+                    DBClusterIdentifier: clusterName,
+                    DBInstanceIdentifier: index === 0 ? clusterName : `${clusterName}${index + 1}`,
+                    DBInstanceClass: result.DBInstanceClass ?? 'db.t3.medium',
+                })
+            }
+        }
```

Comment:
**NIT** - Can this be done concurrently instead of serially using `Promise.all()`?